src/test_workflow/test_cluster.py

```diff
@@ -98,8 +101,8 @@ def __save_test_result_data(self, termination_result: ServiceTerminationResult)
             self.component_name,
             self.component_test_config,
             termination_result.return_code,
-            termination_result.stdout_data,
-            termination_result.stderr_data,
+            ServiceOpenSearch.trans_stdout + termination_result.stdout_data,
+            ServiceOpenSearch.trans_stderr + termination_result.stderr_data,
```

Comment:
Is this supposed to be `ServiceOpenSearch.trans_*` or an instance of the `ServiceOpenSearch` class?

Reply:
What I tried to do was to set-up two new class-level arguments in class ServiceOpenSearch, and then, assign the stdout and stderr when the start() method is triggered. 
Because they are class-level arguments, I can access them by doing like ServiceOpenSearch.trans_stdout.

I actually tried using instence level arguments and applying setter and getter. It did work but I couldn't find the way to Pass the mypy test :(  ; hence, I turned to use class-level arguments instead..  maybe you foresee any issue by doing this way? Thanks!